### ISSUE TITLE
Folders pinned to Start have a gray background

### ISSUE THREAD

**Issue by winston-de** [files-community/Files#4334]
**Explain the bug**
This is likely due to the contrast ratio being too low, so Windows adds a background to make it appear clearly. To fix this, I believe we will need to make the icon used for the tile darker, or use a separate image for light mode.

**To reproduce**
1. Pin folder to start
2. Switch to light theme
3. Tiles have gray background

**Expected behavior**
Tile should have transparent background.

**Screenshots**
![image](https://user-images.githubusercontent.com/59544401/112735978-90ffe100-8f0c-11eb-8040-f84c03954984.png)

**System Information**
 - OS Version: [e.g. Windows 10 20H2 19042.804] (Press WIN + PAUSE/BREAK, use the copy button at the bottom)
 - App version: [e.g. v1.0] (In the app, go to Settings -> About, the version is shown at the top)

**Additional comment**
You can go ahead and assign this to me.

**Comment by timsu92** [comment]
@winston-de May I know the way you pinned?

**Comment by winston-de** [comment]
@timsu92 When you right click a folder while holding the shift key, there will be an option to pin it to start.
![image](https://user-images.githubusercontent.com/59544401/112744307-ced42800-8f53-11eb-9b59-8b0e3171dd3b.png)

**Comment by timsu92** [comment]
Oh~ Thanks!
